ggshield/cmd/config/config_list.py

```diff
@@ -32,7 +33,7 @@ def config_list_cmd(ctx: click.Context) -> int:
 
         message_lines = [
             f"[{instance_name}]",
-            f"default_token_lifetime: {instance.default_token_lifetime}",
+            f"default_token_lifetime: {instance.default_token_lifetime or default_token_lifetime}",
```

Comment:
Not sure about this: isn't `0` a valid value that is considered as "no expiration"? In that case we would need:

```
            f"default_token_lifetime: {instance.default_token_lifetime if instance.default_token_lifetime is not None else default_token_lifetime}",
```